data/json/construction.json

```
@@ -1703,6 +1703,17 @@
     "pre_special": "check_empty",
     "post_terrain": "f_bed"
   },
+  {
+    "type": "construction",
+    "description": "Build Bunk Bed",
+    "category": "FURN",
+    "required_skills": [ [ "fabrication", 4 ] ],
+    "time": "130 m",
+    "qualities": [ [ { "id": "HAMMER", "level": 2 } ], [ { "id": "SAW_W", "level": 1 } ] ],
+    "components": [ [ [ "2x4", 30 ] ], [ [ "nail", 30 ] ], [ [ "mattress", 2 ], [ "down_mattress", 2 ] ] ],
```

```suggestion
    "components": [ [ [ "2x4", 16 ], [ "wood_panel", 4 ] ], [ [ "2x4", 14 ] ], [ [ "nail", 30 ] ], [ [ "mattress", 2 ], [ "down_mattress", 2 ] ] ],
```
This lets them choose to use 2x4 or wood panels for the slats of the bed. Variety is the spice of life!